rest-api/test/unit_test/api_test/questionnaire_response_api_test.py

```diff
@@ -19,20 +20,50 @@ def _questionnaire_response_url(participant_id):
 
 class QuestionnaireResponseApiTest(FlaskTestBase):
 
-  def test_insert(self):
+  def test_insert_raises_400_for_excessively_long_valueString(self):
     participant_id = self.create_participant()
+    questionnaire_id = self.create_questionnaire('questionnaire1.json')
+    url = _questionnaire_response_url(participant_id)
+
+    with open(data_path('questionnaire_response3.json')) as fd:
+      resource = json.load(fd)
+
+    resource['subject']['reference'] = \
+        resource['subject']['reference'].format(participant_id=participant_id)
+
+    resource['questionnaire']['reference'] = \
+        resource['questionnaire']['reference'].format(questionnaire_id=questionnaire_id)
+
+    answer = resource['group']['group'][0]['group'][0]['question'][0]['answer'][0]
```

Comment:
Rather than reading in a questionnaire response file and mutating it in this awkward fashion, let's try using:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/test/unit_test/unit_test_util.py#L572

And passing in a single string_answer that's long.